test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_condition.feature

```diff
@@ -537,49 +538,49 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
       | Content-Type       | application/json                |
   # These properties below are used in subscriptions request
     And properties to subscriptions
-      | parameter               | value                   |
-      | subject_type            | room                    |
-      | subject_idPattern       | .*                      |
-      | condition_attributes    | temperature             |
-      | condition_expression    | q>>><q>                 |
-      | notification_callback   | http://localhost:1234   |
-      | notification_attributes | temperature             |
-      | expires                 | 2016-04-05T14:00:00.00Z |
+      | parameter             | value                   |
+      | subject_type          | room                    |
+      | subject_idPattern     | .*                      |
+      | condition_attrs       | temperature             |
+      | condition_expression  | q>>><q>                 |
+      | notification_http_url | http://localhost:1234   |
+      | notification_attrs    | temperature             |
+      | expires               | 2016-04-05T14:00:00.00Z |
     When create a new subscription
-    And verify that the subscription is stored in mongo
     Then verify that receive an "Bad Request" http code
     And verify an error response
       | parameter   | value                    |
       | error       | BadRequest               |
       | description | invalid query expression |
     Examples:
-      | q               |
-      | speed>=69..90   |
-      | speed>69..90    |
-      | speed<=69..79   |
-      | speed<99..190   |
-      | speed==99...190 |
-      | speed==99,,,190 |
+      | q                |
+      | speed>=69..90    |
+      | speed>69..90     |
+      | speed<=69..79    |
+      | speed<99..190    |
+      | speed==99....190 |
+      | speed==99,,,190  |
 
```

Comment:
Yeah, not **too** surprising, that is case 2. This subscription is useless as the 'q' will never evaluate to TRUE ...

The broker should probably detect this type of problem and return an error ...
@fgalan: your opinion?